operator/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>keycloak-parent</artifactId>
+        <groupId>org.keycloak</groupId>
+        <version>17.0.0-SNAPSHOT</version>
+    </parent>
+    
+    <name>Keycloak Quarkus Operator</name>
+    <artifactId>keycloak-quarkus-operator</artifactId>
+
+    <properties>
+        <!--
+            Override versions based on Quarkus dependencies.
+            Make sure to update these dependencies when Quarkus version changes.
+            See https://github.com/quarkusio/quarkus/blob/<versionTag>/bom/application/pom.xml
+            for reference
+        -->
+        <resteasy.version>4.7.4.Final</resteasy.version>
+        <wildfly.common.version>1.5.4.Final-format-001</wildfly.common.version>
+
+        <compiler-plugin.version>3.8.1</compiler-plugin.version>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <quarkus.operator.sdk.version>3.0.0-SNAPSHOT</quarkus.operator.sdk.version>
+        <quarkus.version>2.6.1.Final</quarkus.version>
```

Comment:
Is there currently any reason to override the version that we use in the project? Is the SDK compatible with it?

Reply:
we are bumping the patch version from what is provided:
https://github.com/quarkiverse/quarkus-operator-sdk/blob/355ac8d8798d3485acffc70133a7dcce04419827/build-parent/pom.xml#L20

Reply:
@andreaTP Could you please mark this as resolved?